server/migrations/20210611155614_digest_mail_user.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = async function(knex) {
+
+    await knex.schema.table('alert', table => {
+        table.datetime('last_email_sent');
```

Comment:
maybe set initial value when the migration is run of all existing alerts to now so a ton of emails won't be sent with old plans?

Reply:
yes i'll add it